cli/scorecard/violations.go

```diff
@@ -67,20 +95,40 @@ func addDataFromBucket(config *ScoringConfig, bucket *storage.BucketHandle, obje
 func getViolations(inventory *InventoryConfig, config *ScoringConfig) (*validator.AuditResponse, error) {
 	v := config.validator
 
-	ctx := context.Background()
-	client, err := storage.NewClient(ctx)
-	if err != nil {
-		return nil, err
-	}
+	if inventory.gcsBucket != "" {
```

Comment:
Instead of using an `if` block here, I'd suggest we refactor this slightly (especially in light of duplication in `addDataFromBucket` and `addDataFromFile`). I'd suggest we have:

1. getViolations calls either `addDataFromBucket` or `addDataFromDirectory`)
2. `addDataFromBucket` and `addDataFromDirectory` contain most of the logic currently implemented here and both call `addDataFromReader`.

Reply:
addressed in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/314/commits/04fb6f7898a992131c4801e1c2e5d5996eeff114